streamwebs_frontend/streamwebs/models.py

```diff
@@ -47,6 +47,21 @@ def __str__(self):
         return self.site_name
 
 
+@python_2_unicode_compatible
+class School(models.Model):
+    name = models.CharField(max_length=250)
+    school_type = models.CharField(max_length=250)
+    address = models.CharField(max_length=250, blank=True)
+    city = models.CharField(max_length=250, blank=True)
+    zipcode = models.CharField(max_length=250, blank=True)
+
+    created = models.DateTimeField(auto_now_add=True)
```

Comment:
Change created and modified to ``models.DateTimeField(default=timezone.now)``

Since there are already preexisting create/mod dates for schools, we shouldn't set ``auto_now_add`` to true.